src/index.js

```diff
@@ -1,22 +1,98 @@
+import Resource from './resources/Resource'
 import Users from './resources/Users'
 import Workplaces from './resources/Workplaces'
+import Cart from './resources/Cart'
 
 const API_HOST = 'backoffice.2cnnct.com'
 const API_VERSION = 1
+const API_LANG = 'en-gb'
+const API_RESELLER_ID = 10000
 
 const resources = {
-  Users,
-  Workplaces
+  users: Users,
+  workplaces: Workplaces,
+  cart: Cart
 }
 
 export default class Deskbookers {
-  constructor (https = true, apiHost = API_HOST, apiVersion = API_VERSION) {
-    this.api = {
-      https, host: apiHost, version: API_VERSION
-    }
+  constructor ({
+    https = true,
+    host = API_HOST,
+    version = API_VERSION,
+    lang = API_LANG,
+    resellerId = API_RESELLER_ID
+  }) {
+    // Options
+    this.https = https
+    this.host = host
+    this.version = version
+    this.lang = lang
+    this.resellerId = resellerId
+    this.session = null
 
+    // Init resources
     for (let name in resources) {
-      this[`${name.toLowerCase()}`] = new resources[name](this.api)
+      this[name] = new resources[name](this)
+    }
+  }
+
+  hasSession () {
+    return !!(this.session && this.session.privateKey)
+  }
+
+  getSession () {
```

Comment:
I refactored setSession and getSession to setters and getters. I didn't inline the line because I think it's more readable this way.